siliconcompiler/schema/docs/schemagen.py

```
@@ -84,13 +84,15 @@ def run(self):
 
             # Add docstrings
             docstring = None
+            docsfile = None
             for mro_cls in schema_cls.mro():
```

Ruff (0.13.1)

Loop control variable `mro_cls` not used within loop body

(B007)